core/lib/storage/src/db.rs

```diff
@@ -329,6 +329,25 @@ impl<CF: NamedColumnFamily> RocksDB<CF> {
         // We panic on RocksDB errors elsewhere and fuse it to prevent polling after the end of the range.
         // Thus, `unwrap()` should be safe.
     }
+
+    /// Iterates over key-value pairs in the specified column family `cf` in the lexical
+    /// key order starting from the given `key_from`.
+    pub fn from_iterator_cf(
+        &self,
+        cf: CF,
+        key_from: &[u8],
+    ) -> impl Iterator<Item = (Box<[u8]>, Box<[u8]>)> + '_ {
+        let cf = self.column_family(cf);
+        self.inner
+            .db
+            .iterator_cf(cf, IteratorMode::From(key_from, Direction::Forward))
+            .map(Result::unwrap)
+            .fuse()
+        // ^ The rocksdb docs say that a raw iterator (which is used by the returned ordinary iterator)
+        // can become invalid "when it reaches the end of its defined range, or when it encounters an error."
+        // We panic on RocksDB errors elsewhere and fuse it to prevent polling after the end of the range.
+        // Thus, `unwrap()` should be safe.
```

Comment:
Bikeshedding: instead of copy-pasting, I'd refer to the above comment: "`unwrap()` is safe for the same reasons as in `prefix_iterator_cf()`".